augur/tree.py

```diff
@@ -32,7 +32,10 @@
     # with -fast, although there's probably no particular reason we have to.
     # Refer to the handling of -fast in utils/tools.cpp:
     #   https://github.com/Cibiv/IQ-TREE/blob/44753aba/utils/tools.cpp#L2926-L2936
-    "iqtree": "-ninit 2 -n 2 -me 0.05",
+    # Increasing threads (nt) can cause IQtree to run longer, hence use AUTO by default
+    # Auto makes IQtree chose the optimal number of threads
+    # Redo prevents IQtree errors when a run was aborted and restarted
+    "iqtree": "-ninit 2 -n 2 -me 0.05 -nt AUTO -redo",
```

Comment:
For the reason to add `-redo` see #1037

Reply:
Ah, this is really helpful for cases where `augur tree` replaces the offending characters in strain names with random strings. The random strings will always cause an error when iqtree uses checkpoints.